usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -1,6 +1,65 @@
 DROP VIEW IF EXISTS award_delta_view;
-
 CREATE VIEW award_delta_view AS
+with closed_periods as (
+    select
+    	distinct concat(w.submission_fiscal_year::text, lpad(w.submission_fiscal_month::text, 2, '0')) AS fyp,
```

Comment:
Can get rid of this line now. See updated CTE sql in sql files attached to https://federal-spending-transparency.atlassian.net/browse/DEV-5283
- it gets rid of messing around with left-padding